### DISCUSSION TITLE
[SEO优化] notion表格加入针对SEO的列

### DISCUSSION THREAD

**Discussion by bjtiew-gtech** [tangly1024/NotionNext#2064] [Ideas]
**为什么提出这个新的特性改动**
进一步优化SEO功能

**描述一下你推荐的解决方案**
参考了 https://wr8.in/ 的方案，即在notion表格里加上下列column：
1. MetaDescription
2. SEOCreatedDate
3. SEOUpdateDate
4. SEOTitle

**描述一下你考虑过的其它替代解决方案**
新手上路，储备知识几乎为零，没啥想法

**补充说明**
无

**Comment by tangly1024** [comment]
感谢反馈，很不错的提议

**Comment by raygorous** [comment]
这个有什么进展吗？同意 @BJTiew 的观点。以下是我用SEO optimizer run自己的网站 [raygorous.com](https://www.raygorous.com) 跑出来的结果：

Keyword Optimization: The analysis didn't find a specific keyword focus for your website. It's crucial to identify and use relevant keywords in your website's content, meta tags, and titles. This will help search engines understand what your site is about and improve its ranking for those keywords.

Title and Meta Description: Your website's title is quite long with 124 characters. It's recommended to keep it under 60 characters. Similarly, your meta description is 101 characters long. Try to keep it under 160 characters. Both should be concise and contain your focus keyword(s).

H1 Tags: Make sure your H1 tags are descriptive and contain your focus keyword(s). They should provide a clear description of the page's content.

**Comment by donttal** [comment]
想问一下这个有后续吗？想做网站的seo优化

**Comment by real-jacket** [comment]
+1，现在博客的文章基本搜索不到，希望能有这方面的优化，以及相应的稳定操作
